src/Umbraco.Web.UI.Client/src/common/directives/components/content/edit.controller.js

```diff
@@ -53,7 +52,117 @@
+
+            //If there are no editors yet then create one with the current content.
+            //if there's already a main editor then update it with the current content.
+            if ($scope.editors.length === 0) {
+                var editor = {
+                    content: activeVariant
+                };
+                $scope.editors.push(editor);
+            }
+            else {
+                //this will mean there is only one
+                $scope.editors[0].content = activeVariant;
+
+                if ($scope.editors.length > 1) {
+                    //now re-sync any other editor content (i.e. if split view is open)
+                    for (var s = 1; s < $scope.editors.length; s++) {
+                        //get the variant from the scope model
+                        var variant = _.find($scope.content.variants, function (v) {
+                            return v.language.culture === $scope.editors[s].content.language.culture;
+                        });
+                        $scope.editors[s].content = initVariant(variant);
+                    }
+                }
+                
+            }
+        }
+
+        function initVariant(variant) {
+            //The model that is assigned to the editor contains the current content variant along
+            //with a copy of the contentApps. This is required because each editor renders it's own
+            //header and content apps section and the content apps contains the view for editing content itself
+            //and we need to assign a view model to the subView so that it is scoped to the current
+            //editor so that split views work. 
+
+            //copy the apps from the main model if not assigned yet to the variant
+            if (!variant.apps) {
+                variant.apps = angular.copy($scope.content.apps);
+            }
+
+            //if this is a variant has a culture/language than we need to assign the language drop down info 
+            if (variant.language) {
+                //if the variant list that defines the header drop down isn't assigned to the variant then assign it now
+                if (!variant.variants) {
+                    variant.variants = _.map($scope.content.variants,
+                        function(v) {
+                            return _.pick(v, "active", "language", "isEdited", "state");
+                        });
+                }
+                else {
+                    //merge the scope variants on top of the header variants collection (handy when needing to refresh)
+                    angular.extend(variant.variants,
+                        _.map($scope.content.variants,
+                            function(v) {
+                                return _.pick(v, "active", "language", "isEdited", "state");
+                            }));
+                }
+            }
+
+            //ensure the current culture is set as the active one
```

Comment:
I think this was supposed to be inside of ```if (variant.language)``` above, as if an invariant node is created, ```variant.variants``` will be undefined.

Reply:
indeed, have pushed a fix for that